src/commands/createFunctionApp/stacks/getStackPicks.ts

```diff
@@ -176,3 +197,9 @@ function removeHiddenStacksAndProperties(stacks: FunctionAppStack[]): void {
         }
     }
 }
+
+export function compareDates(eolDate: string, sixMonthsDate: Date): boolean {
+    const endOfLife = new Date(eolDate).getTime();
+    const sixMonths = new Date(sixMonthsDate).getTime();
+    return endOfLife <= sixMonths;
```

Comment:
You can compare `Date` objects with `<=` directly in JS without having to do `.getTime()`